### PR TITLE
Sagnik17 - Add introduction files

### LINKED ISSUE #1: sagnik17 Workflow
## Sagnik Saha will update a file by replacing a caption.

- [x] Create a branch
- [x] Edit the file
- [x] Commit the changes
- [x] Create a pull request
- [x] Request a review
- [x] Make more changes
- [x] Get an approval
- [x] Merge the pull request

### PR DESCRIPTION
**Pull request by sagnik17**
This PR contains the intro files about Sagnik.

Changes Made:
- Added a Sagnik.md file.
- Added a fun fact file.
- Create a file puppypic.md

Fixes #1

### CODE REVIEW

File: Sagnik17.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# My name is Sagnik Saha
+# Nickname - Rick
```
Comment (JumboTonic) [inline Sagnik17.md:2]: Rick! I didn't know this :)

**Review by JumboTonic — APPROVED** [review]
pass

**Review by charlie1682025 — APPROVED** [review]
looks good,

**Review by garyei — APPROVED** [review]

**Comment by garyei** [comment]
Approved

**Pull request merged**